docs/articles/roadmap/README.md

```diff
@@ -15,28 +16,31 @@ Please vote on the features you want us to implement first to help us prioritize
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **In Progress**
-[API Testing](https://github.com/DevExpress/testcafe/issues/4288)                                 | Planned
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **Implemented**
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | **In Progress**
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
+[Cookie Management](https://github.com/DevExpress/testcafe/issues/5964)                           | Planned
 
 ## API
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Configuration File](https://github.com/DevExpress/testcafe/issues/3131)                          | **Implemented**
-[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | **Implemented**
-[Browser Information in Test Code](https://github.com/DevExpress/testcafe/issues/481)             | **Implemented**
+[Dedicated *Scroll* Action](https://github.com/DevExpress/testcafe/issues/5965)                   | Planned
 
 ## Debugging
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Video Recording](https://github.com/DevExpress/testcafe/issues/2151)                             | **Implemented**
-[Live Mode](https://github.com/DevExpress/testcafe/issues/3215)                                   | **Implemented**
-[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | **In Progress**
+[Interactive Demo Center](https://github.com/DevExpress/testcafe/issues/5966)                     | Planned
 
-## Ecosystem / Integrations
+## Test Stability
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Custom TypeScript Compilation Options](https://github.com/DevExpress/testcafe/issues/1845)       | **Implemented**
-[Jenkins Plugin](https://github.com/DevExpress/testcafe/issues/4289)                              | **Implemented**
+[Retry Failed Requests for Tested Webpages](https://github.com/DevExpress/testcafe/pull/5738)     | **Implemented**
+[HTTP Caching](https://github.com/DevExpress/testcafe-hammerhead/issues/863)                      | **In Progress**
+[Improve Waiting Mechanism for Assertions](https://github.com/DevExpress/testcafe/issues/5967)    | Planned
```

Comment:
please please please let's stop using the word "mechanism", native English speakers don't use it the way Russian speakers do.

```suggestion
[Improve Assertion Waiting Algorithm](https://github.com/DevExpress/testcafe/issues/5967)    | Planned
```

Reply:
I agree, `mechanism` isn't perfect, but we [use it a lot ](https://devexpress.github.io/testcafe/documentation/guides/concepts/built-in-wait-mechanisms.html)in our docs. I'd leave it as is for consistency (until we reconsider the term, which we should).
`Algorithm` doesn't fit the context in my opinion

Reply:
a) could you please elaborate re: 'algorithm'? 
here's the Oxford dictionary [definition](https://www.oxfordlearnersdictionaries.com/definition/english/algorithm) of the word: 
`​a set of rules that must be followed when solving a particular problem`
what are we going to improve if not the steps that are being followed when determining the timeout value?

b) we can just leave it out if you still don't agree:

```suggestion
[Improve the Quality of Automatic Pre-Assertion Timeouts](https://github.com/DevExpress/testcafe/issues/5967)    | Planned
```